MdePkg/Library/BasePeCoffLib2/PeCoffHash.c

```diff
@@ -19,7 +19,6 @@
 #include <Library/BaseOverflowLib.h>
 #include <Library/DebugLib.h>
 #include <Library/MemoryAllocationLib.h>
-#include <Library/PcdLib.h>
```

Comment:
@MikhailKrichanov, I am afraid you removed a bit too much. This file uses PcdLib directly, and thus the include should stay here. All direct interface usages (i.e. when a .c file executes an interface from a Lib.h file) must be covered by direct includes and not rely on some other header implicitly providing the interface for them.

You might have removed way too much for other .c files in the BasePeCoffLib2 as well, could you please recheck?